src/main/java/mate.academy/service/FileReaderService.java

```diff
@@ -3,5 +3,6 @@
 import java.util.List;
 
 public interface FileReaderService {
+    @Component
```

Comment:
There is no need to mark the method. With @Component we need to mark the class which implements interface.

Reply:
done